napari/_qt/menus/help_menu.py

```diff
@@ -12,17 +18,59 @@
 except ModuleNotFoundError:
     ask_opt_in = None
 
+# Get the version for proper links to documentation
+v = parse(__version__)
+VERSION = "dev" if v.is_devrelease else str(v)
+
 
 class HelpMenu(NapariMenu):
     def __init__(self, window: 'Window'):
         super().__init__(trans._('&Help'), window._qt_window)
         ACTIONS = [
             {
-                'text': trans._('napari Info'),
+                'text': trans._('Getting started'),
+                'slot': lambda e: webbrowser.open(
+                    f'https://napari.org/{VERSION}/tutorials/start_index.html'
+                ),
+                'statusTip': trans._('Open Getting started webpage'),
+            },
+            {
+                'text': trans._('Tutorials'),
+                'slot': lambda e: webbrowser.open(
+                    f'https://napari.org/{VERSION}/tutorials/index.html'
+                ),
+                'statusTip': trans._('Open Tutorials webpage'),
+            },
+            {
+                'text': trans._('Examples Gallery'),
+                'slot': lambda e: webbrowser.open(
+                    f'https://napari.org/{VERSION}/gallery.html'
+                ),
+                'statusTip': trans._('Open Examples Gallery webpage'),
+            },
+            {
+                'when': VERSION != "dev",
+                'text': trans._('Release Notes'),
+                'slot': lambda e: webbrowser.open(
+                    f'https://napari.org/{VERSION}/release/release_{VERSION.replace(".", "_")}.html'
+                ),
+                'statusTip': trans._('Open Release Notes webpage'),
+            },
+            {
+                'text': trans._('napari homepage'),
+                'slot': lambda e: webbrowser.open('https://napari.org'),
+                'statusTip': trans._('Open napari.org webpage'),
+            },
+            {},
+            {
+                'text': trans._('napari info'),
```

Comment:
Should the napari info stay at the top of the help menu? We ask people to submit it whenever they report bugs, so might be good to keep it easy to find

Reply:
> I swapped napari info to the bottom of the menu after a divider—this is typical on Windows, though the name is About ... not ... info. Plus, I think it makes sense to have the Help at the top. I also added the Release notes, if the version isn't dev. I think easy access to the release notes is handy, plus it seems typical in other software (e.g. VS Code)

Ah, I see that was a deliberate choice. Ok, so long as we tell people where to find it when they submit bug reports that should be fine.